app/pages/Home/component.jsx

```diff
@@ -12,8 +12,8 @@ export default ({ submitFile, submitURL, redirectTo }) => (
             <Col md={12}>
                 <p style={{ fontSize: '18px' }}>
                     This interface can view .qza and .qzv files
-                    directly in your browser without uploading to a server.
-                    <Link to="/about">Click here</Link> to learn more.
+                    directly in your browser without uploading to a server. <Link to="/about">Click
+                        here</Link> to learn more.
```

Comment:
This missing space has been bugging me for a while! What if we address it with something like this, using an HTML entity (and some addition markup):

```suggestion
                    directly in your browser without uploading to a server.
                    <span>&nbsp;<Link to="/about">Click here</Link> to learn more.</span>
```

PS, I think I fixed up whatever was going on with the travis failures... 🤞